pandas/core/frame.py

```diff
@@ -1815,13 +1815,17 @@ def info(self, verbose=None, buf=None, max_cols=None, memory_usage=None,
         """
         Concise summary of a DataFrame.
 
+        This method shows information about DataFrame type of index, columns
+        dtypes, non-null values and memory usage.
```

Comment:
I'd like to recommend some little changes in the text: "This method shows information about a DataFrame: index dtype, ..."

Reply:
Done!